spec/controllers/forms_controller_spec.rb

```diff
@@ -139,6 +143,17 @@
             submit
             expect(response).to redirect_to thank_you_form_url(@form)
           end
+          it 'updates the current_user object' do
+            [:email, :first_name, :last_name].each do |attribute|
+              expect(@current_user.send(attribute))
```

Comment:
I like not being repetitive, but do you think the only way to make this clearer is to run a test for each attribute?